18.04/php7.2/Dockerfile

```diff
@@ -82,32 +88,41 @@ RUN apt install -y \
 
 # Install Composer and Drush
 RUN curl -sS https://getcomposer.org/installer | php -- \
-    --install-dir=/usr/bin \
+    --install-dir=/usr/local/bin \
     --filename=composer \
     --version=1.7.3 && \
     composer \
     --working-dir=/usr/local/src/ \
     global \
     require \
     drush/drush:8.* && \
-    ln -s /usr/local/src/vendor/bin/drush /usr/bin/drush
+    ln -s /usr/local/src/vendor/bin/drush /usr/local/bin/drush
 
 # Download Drupal Console and install it in our standard path.
-RUN curl https://drupalconsole.com/installer -L -o /drupal.phar
-RUN mv /drupal.phar /bin/drupal
-RUN chmod 755 /bin/drupal
+RUN curl https://drupalconsole.com/installer -L -o /tmp/drupal.phar
+RUN mv /tmp/drupal.phar /usr/local/bin/drupal
+RUN chmod +x /usr/local/bin/drupal
 
 # Get Node Package Manager
 RUN apt install -y npm
 
-# Install Proboscis
+# Install Proboscis.
+# This package is required!
 RUN npm install -g proboscis \
   && ln -s /usr/lib/node_modules/proboscis/bin/proboscis /usr/bin/proboscis \
   && ln -s /usr/lib/node_modules/proboscis/bin/proboscis-control /usr/bin/proboscis-control \
   && ln -s /usr/lib/node_modules/proboscis/bin/proboscis-format /usr/bin/proboscis-format \
   && rm -rf /var/lib/apt/lists/* /tmp/*
 
-# Start and configure our PostgreSQL server
+# Install NPM Packages.
+RUN npm install -g bower \
+  && grunt-cli \
+  && gulp-cli \
+  && lighthouse \
+  && the-a11y-machine \
+  && yarn
```

Comment:
I fixed this on my local branch, but this will fail because you are && these packages. They should have the 'npm install' prefixed, or removed.

Reply:
Woops, that was silly! I'll fix these and merge these changes in so we can start testing out these images.

Reply:
I've fixed these lines in this PR and added the npm install steps for these tools too the other 18.04 PHP images.